### DISCUSSION TITLE
Keep target spotlight with palacement = center

### DISCUSSION THREAD

**Discussion by francescocretti** [gilbarbara/react-joyride#1021] [Q&A] [answered]
Hi everyone.
Is it possible to use the `placement: "center"` prop while keeping the element spotlight (I mean the back overlay with the transparent frame only on the target element) on? 

My goal is to highlight the current step's target, but since the tutorial content is pretty big, I want to show it in a centered overlay.

Currently, if I use `placement: "center"` the target is ignored.

Hope I explained myself.

Thanks in advance.

**Comment by gilbarbara** [comment, marked as answer]
Hey @francescocretti 

It's not currently possible.
If you set the step's `target` to an HTML Element, the tooltip will try to position itself relative to it instead of being centered.

But I'll took at look at the code again to see if there's something that could be done.

**Reply by gilbarbara** [reply]
So, I've tried to show the spotlight with the placement center, and it doesn't look good. Feels like something is broken.
So I won't add support for it right now.

**Reply by francescocretti** [reply]
ok @gilbarbara thanks anyways for your time
